CONTRIBUTING.md

```diff
@@ -79,29 +66,31 @@ nx reset
+2. Nx will notice what packages have been affected by your changes and ask you what kind of version bump is required. Then it'll ask for a changelog message. From that, it'll create a version plan file. Commit that and include it as part of your PR.
+3. When a PR is merged which includes a version plan, a release PR will be automatically opened to move the changelog entries into the changelogs and release the new versions. That PR will update as more features are merged to roll up all pending changes. It'll also bump any packages which depend on changed packages to point to the latest version.
+4. When _that_ PR is merged, those new versions will be published, and releases will be added to the GitHub repo.
 
-### What's a Release PR?
+Note that there's a single release PR for all packages in the repo (or no PR, if there's nothing to publish right now). This ensures that we always publish the versions that depend on one another all together.
 
-Rather than continuously releasing what's landed to our default branch, release-please maintains Release PRs:
+### How do we define our version bumps?
 
-These Release PRs are kept up-to-date as additional work is merged. When we're ready to tag a release, we simply merge the release PR.
+We use [SemVer](https://semver.org/) for our versioning scheme. Quoting from there:
 
-When the release PR is merged the release job is triggered to create a new tag, a new github release and run other package specific jobs. Only merge ONE release PR at a time and wait for CI to finish before merging another.
+> Given a version number MAJOR.MINOR.PATCH, increment the:
+>
+> 1. MAJOR version when you make incompatible API changes
+> 2. MINOR version when you add functionality in a backward compatible manner
+> 3. PATCH version when you make backward compatible bug fixes
 
-Release PRs are created individually for each package in the mono repo.
+That is, if package depends on another package's version `2.3.4`, the first package should:
 
-### How should I write my commits?
+1. Be able to use any version `2.3.x` without changes to its code, though earlier releases may be buggier than later ones.
+2. Be able to use any version `2.x.y` without changes to its code _as long as `x` is at least `3`_, because it depends on features that were released in `2.3` and weren't available in `2.2`.
+3. _Not_ know whether it can use any version that doesn't start with `2` without changing its code. Version `1` and version `3` may have completely incompatible APIs.
 
-Release Please assumes you are using [Conventional Commit messages](https://www.conventionalcommits.org/).
+The release system will always create the smallest correct version bump. So, if a package is currently at `2.3.4`, given three `minor` changes, it would bump to `2.4.0`, rolling all three into a single minor-version bump. But given those three `minor` changes _and_ a `major` (ie, breaking) change, it would bump to `3.0.0`.
```

Comment:
How does this work with multiple feature/fix PRs that we want to bundle into a single release?

Reply:
That's exactly what this is describing. They'll result in the smallest correct bump that covers all of them.